lib/london_bike_hire_cli/query_response.rb

```diff
@@ -1,20 +1,12 @@
-require 'forwardable'
+require 'delegate'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.